pom.xml

```diff
@@ -83,6 +83,17 @@
             <groupId>org.elasticsearch</groupId>
             <artifactId>elasticsearch</artifactId>
             <version>${es.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
+                    <artifactId>log4j-api</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
@rajdangwal This exclusion is redundant, if we are pinning a dependency explicitly. that will override the transitive dependency.